requirements/constraints.txt

```diff
@@ -10,3 +10,5 @@
 
   # These packages are backports which can only be installed on Python 2.7
 futures ; python_version == "2.7"
+more-itertools<6 
+pytest==4.6.7
```

Comment:
These two should probably get their own comments for why they've been constrained.